packages/quantic/force-app/main/default/lwc/quanticPager/e2e/pagerObject.ts

```diff
@@ -0,0 +1,69 @@
+import {Locator, Page, Request} from '@playwright/test';
+import {isUaCustomEvent} from '../../../../../../playwright/utils/requests';
+
+export class PagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get nextPageButton(): Locator {
+    return this.page.locator('button[title="Next Page"]');
```

Comment:
I think we could do a better job of using the locator from Playwright too:


```suggestion
    return this.page.getByRole('button', { name: /Next Page/i });
```

Not sure if the `name` works well, otherwise you can use:

```
return this.page.getByRole('button').filter({ hasText: /Next Page/i });
```